phoenix-core/src/it/java/org/apache/phoenix/end2end/index/ImmutableIndexIT.java

```diff
@@ -17,10 +17,15 @@
  */
 package org.apache.phoenix.end2end.index;
 
+import static org.apache.phoenix.end2end.IndexToolIT.assertExplainPlan;
+import static org.apache.phoenix.schema.types.PDataType.FALSE_BYTES;
+import static org.apache.phoenix.schema.types.PDataType.TRUE_BYTES;
```

Comment:
Instead of using TRUE_BYTES and FALSE_BYTES, please use IndexRegionObserver.UNVERIFIED_BYTES and IndexRegionObserver.VERIFIED_BYTES. I have changed this recently.

Reply:
Will do